src/tests/components/ResetPassword.test.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import { toast } from 'react-toastify';
+import { ResetPasswordPage } from '../../components/resetPassword/ResetPasswordPage';
+
+let wrapper;
+
+describe('ResetPasswordPage component', () => {
+  const createSpy = (toSpy) => jest.spyOn(wrapper.instance(), toSpy);
+  const token = 'dosks';
+  const params = { token };
+  const match = { params };
+
+  const getEvent = (name = '', value = '') => ({
+    preventDefault: jest.fn(),
+    target: {
+      name,
+      value,
+    },
+  });
+
+  const props = {
+    history: { push: jest.fn() },
+  };
+
+  const nextProps = {
+    updatePasswordSuccess: true,
+  };
+
+  const state = {
+    state: {
+      password: {
+        sendLinkSuccess: false,
+        updatePasswordSuccess: false,
+      },
+    },
+  };
+
+  beforeEach(() => {
+    wrapper = mount(
+      <ResetPasswordPage
+        match={match}
+        dispatch={jest.fn}
+        sendLinkSuccess={state}
+        {...props}
+      />,
+    );
+  });
+
+  afterEach(() => {
+    wrapper.unmount();
+  });
+
+  it('should render correctly', () => {
+    expect(wrapper).toMatchSnapshot();
+  });
+
+  it('should call handleChange on resetPassword form fill', () => {
+    const spy = createSpy('handleChange');
+    wrapper.instance().forceUpdate();
+    wrapper.find('input[name="password1"]').simulate('change', { target: { value: 'some value' } });
+    expect(spy).toBeCalled();
+  });
+
+  it('should not accept if passwords are not the same', () => {
```

Comment:
@RonKbS, I do not understand this test. What shouldn't it accept?

Reply:
The test checks if the component rejects the form submission if the second input is invalid.

Reply:
@RonKbS, your response to @kakaemma here seems more descriptive. Please consider using that instead. `it should not submit form data if the passwords are not the same`.